### ISSUE TITLE
Wrong output

### ISSUE THREAD

**Issue by anselal** [Ext-Creators/discord-ext-ipc#42]
I am using the latest version 2.1.0 and the following code returns the wrong value for some reason:

`bot:`
```python
    @ipc.server.route()
    async def get_guild_count(self, data):
        return len(self.bot.guilds)  # returns the len of the guilds to the client
```

`dashboard`
```python
guild_count = await ipc_client.request("get_guild_count")
```

`result:`
```json
{'message': 'Connection success', 'port': '8765', 'code': 200}
```

I didn't change the code, I just updated to the latest version

**Comment by shayypy** [comment]
I believe this occurs when you specify `port` and then the client tries to connect via multicast anyway. #43 should fix this, but you can also not specify a `port` and let the client find it with multicast

**Comment by anselal** [comment]
Will try that and let you know.

**Comment by jgayfer** [comment]
I was able to get around this by not passing a `port` to my `Client`.

**Comment by ShineyDev** [comment]
Closed by #43.

**Comment by anselal** [comment]
Sorry for the late reply. I just removed the port from the client and I confirm that the issue got fixed.
Pretty strange though

thnx 👍
